docs/team/yongmingyang.md

```diff
@@ -9,45 +9,40 @@ Medmoriser is a desktop app to help medical students organise, memorise and revi
 
-* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2021s1.github.io/tp-dashboard/#breakdown=true&search=&sort=groupTitle&sortWithin=title&since=2020-08-14&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other&tabOpen=true&tabType=authorship&tabAuthor=yongmingyang&tabRepo=AY2021S1-CS2103T-W15-1%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code~other)
-
-* **Project management**:
-  * Managed releases `v1.3` - `v1.5rc` (3 releases) on GitHub
+* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2021s1.github.io/tp-dashboard/#breakdown=true&search=yongmingyang)
 
 * **Enhancements to existing features**:
-  * Redefined the Find functionality
+  * Redefined the Find functionality:
     * What it does: Allows the user to find based on 3 modes
         1. `find t/[tagname]` which allows the user to find QAndA based on the tagname(s).
         2. `find q/[keyword]` which allows the user to find QAndA which contains the specified keyword(s) in the question.
         3. `find [keyword]` which allows the user to find QAndA which contains the specified keyword(s) in the question or the answer.
     * Justification: Gives the user more flexibility to find relevant QAndA sets.
-    * Added additional test cases to suit the redefined functionality.
-    * Pull requests/Issues:
-        1. [\#50]()
-        2. [\#56]()
-        3. [\#76]()
-        4. [\#97]()
-        5. [\#103]()
-        6. [\#122]()
-        7. [\#126]()
+    * Added additional test cases to suit the redefined functionality, increased code coverage by 2.56%: [#58](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/58)
+    * Pull requests (inclusive of bug fixes):
+        1. [#55](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/55)
+        2. [#58](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/58)
+        3. [#68](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/68)
+        4. [#75](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/75)
+        5. [#81](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/81)
+        6. [#138](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/138)
+        7. [#139](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/139)
     * Credits: The find functionality makes use of code from the following links:
         1. https://stackoverflow.com/questions/36793397/check-if-string-contains-word-not-substring.
-
+  * Refactoring instances of `Address` to `Answer`, `UniquePersonList` to `UniqueQuestionSetList`, `AddressBook` to `Medmoriser` and `Person` to `QuestionSet` (later changed to QAndA by Jian Ling): [#36](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/36)
+  * Updated GUI, for quizing part in particular: [#165](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/165), [#172](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/172), [#46](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/46)
```

Comment:
Might want to list the PRs in ascending order

Reply:
aight